test/libgit2.jl

```diff
@@ -346,6 +348,7 @@ mktempdir() do dir
                 LibGit2.add!(repo, test_file)
                 status = LibGit2.GitStatus(repo)
                 @test length(status) != 0
+                @test status[0] == nothing
```

Comment:
It's better since it's type independent. It matters less for tests though (but is still preferred).